OHHTTPStubs/Podfile

```diff
@@ -7,7 +7,7 @@ abstract_target 'TestingPods' do
   pod 'AFNetworking', '~> 3.0'
 
   target 'OHHTTPStubs iOS Lib Tests' do
-    platform :ios, '7.0'
+    platform :ios, '8.0'
```

Comment:
We should mention that minimum certain requirement bump in the CHANGELOG too, after all it's a breaking change

Reply:
Updated in c9b28c9.